src/app/Fake.Core.Target/Target.fs

```diff
@@ -328,9 +333,14 @@ module Target =
           |> Seq.filter (fun kv -> kv.Value)     // only if activated
           |> Seq.map (fun kv -> kv.Key)
           |> Seq.fold (fun context name ->
-               Trace.tracefn "Starting BuildFailureTarget: %s" name
-               let target = get name
-               runSimpleContextInternal target context) context
+                        let target = get name
+                        use t = Trace.traceFailureTarget target.Name (match target.Description with Some d -> d | _ -> "NoDescription") (dependencyString target)
```

Comment:
instead of `"NoDescription"` we use `null` now and properly handle that case in the listeners.

A better solution would be to use `string option` (which now is at the table considering the more breaking addition -  see other comment), see https://github.com/fsharp/FAKE/pull/1996

Reply:
Updated and have also made this a string option and fixed all the listeners we have.